WooCommerce/Classes/POS/Presentation/CartView.swift

```diff
@@ -18,38 +18,18 @@ struct CartView: View {
 
     var body: some View {
         VStack {
-            DynamicHStack(spacing: Constants.cartHeaderSpacing) {
-                HStack(spacing: Constants.cartHeaderElementSpacing) {
-                    backAddMoreButton
-                        .disabled(shouldPreventCartEditing)
-                        .shimmering(active: shouldPreventCartEditing)
-
-                    HStack {
-                        Text(Localization.cartTitle)
-                            .font(Constants.primaryFont)
+            POSPageHeaderView(title: Localization.cartTitle,
+                              backButtonConfiguration: backButtonConfiguration,
+                              trailingContent: {
+                DynamicHStack(horizontalAlignment: .trailing, verticalAlignment: .center, spacing: Constants.cartHeaderElementSpacing) {
```

Comment:
🗒️ Note for reviewer: with the new page header component with the title built in the component and separate trailing content, it's not straightforward to use `DynamicHStack` with the previous grouping anymore. I'm also wary of any performance impact from using `AnyLayout` in `DynamicHStack`, if it's similar to `AnyView`. Therefore, I moved `DynamicHStack` to the trailing content (items count label and Clear CTA). Please feel free to share any other improvements.

<img src="https://github.com/user-attachments/assets/15166fe4-6cbc-44ea-bb3e-e53d7d1735a6" width="500" />

Reply:
IMHO, in this specific case, `POSPageHeaderView` would look nicer with `HStack(alignment: .top`. So Cart wouldn't be centered. 

However, when we remove `Clear` button in the Checkout view, then centering rather than the top looks better. 

At this point, we can leave as it is. Most importantly, the layout adapts to larger font sizes.